meta-cube/recipes-support/overc-utils/source/cube-ctl

```diff
@@ -242,11 +261,22 @@ case "${cmd}" in
 	    container_name="`cubename ${filename}`"
 	fi
 
-	${overc_cctl} add -d -a -c -g onboot -t 0 -n ${container_name} -f ${container_source}
+	# unprivileged. create default subuid/subgids for root and pass the
+	# parameter to overc_ctl for container settings
+	if [ ! -z ${subuid} ]; then
+		privilege_param="-p -g peer -u ${subuid}"
+		cube-cmd cmd "touch /etc/subuid /etc/subgid"
+		cube-cmd cmd "usermod --add-subuids ${subuid}-$(( subuid+65536 )) root"
+		cube-cmd cmd "usermod --add-subgids ${subuid}-$(( subuid+65536 )) root"
+	else
+		privilege_param="-g onboot -a"
```

Comment:
Currently the onboot group is started by essential, somehow we now manage cgroup settings within dom0. So these unprivileged containers cannot be started if dom0 is not ready yet.